content/blog/2023/01/release-1_5_0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Kubewarden 1.5.0 release
+authors:
+- José Guilherme Vanz
+date: 2023-01-18
+---
+
+Today we're pleased to announce the availability of Kubewarden 1.5.0!
+
+This release brings the usual amount of small bug fixes, dependency updates and a major security feature.
+Let's take a closer look!
+
+## Policy evaluation timeout
+
+The Kubewarden team is constantly working to improve the security posture of the project. As part of this efforts,
+we're excited to introduce the new "policy evaluation timeout" feature.
+
+Starting from this release, Policy Server will interrupt the evaluation of admission requests after a certain amount of time has elapsed.
+This security feature, which is enabled by default, prevents a Policy Server from running out of computing
+resources because one or more of its policy evaluations are stuck in infinite loops.
+This mitigates a type of Denial Of Service (DOS) attacks against the Policy Server.
+
+Take a look in the Kubewarden documentation to get more information about this new feature.
+
+## Changes to kwctl
+
+`kwctl` now imports the trusted certificate authorities from from the host system. Thus, `kwctl`
+interactions with registries secured by certificates issued by 3rd party certificate authorities
+becomes simpler.
+
+Note, it's still possible to fine tune the certificates to be used via the [`sources.yml` file](https://docs.kubewarden.io/distributing-policies/custom-certificate-authorities#the-sourcesyaml-file).
+
+## Changes to Kubewarden controller
+
+The validation of `ClusterAdmissionPolicy` and `AdmissionPolicy` custom resources has been extended. The
+new validation checks are able to prevent the creation of resources that do not have any `rule` specified.
+
+We would like to thank [Mauren Berti](https://github.com/stormqueen1990) for this contribution.
```

Comment:
Mauren's a friend and I mean her no harm, but she is an employee of SUSE now. I'm not sure if we should be singling out folks unless they're community members like this.

Reply:
@stormqueen1990, are you okay being cited here ?

Reply:
I share @divya-mohan0209's concern. I'd prefer not being nominally cited.

Reply:
I agree with @divya-mohan0209. I've removed the link